cli/fs.js

```diff
@@ -54,7 +81,13 @@ export async function loadFs(args) {
   }
 
   if (hasPrompt) {
+    if (edit && content.length == 1) {
+      edit.file = content[0];
+      content = [];
+    }
     const noContext = args.values.context == "none";
+    const folder = args.values.context == 'folder' ?
+      Object.values(context).find(c => dirname(c.path) == root)?.context.folder : edit ? [edit.file] : undefined;
```

Comment:
Ok, so no start/end means `context.folder === undefined`

Reply:
if context is folder, use the whole folder; otherwise, if editing, folder is the one file; otherwise, no folder.